test/modules/generative-palm/generative_palm_test.go

```diff
@@ -121,9 +124,26 @@ func testGenerativePaLM(host, gcpProject string) func(t *testing.T) {
 					planets.CreateTweetTest(t, class.Class)
 				})
 				t.Run("create a tweet with params", func(t *testing.T) {
-					params := "google:{topP:0.1 topK:40}"
+					params := "palm:{topP:0.1 topK:40}"
```

Comment:
I can see still here that you are using `palm` namespace instead of `google`

Reply:
Thank you for spotting it!